ambry-replication/src/main/java/com/github/ambry/replication/ReplicationManager.java

```diff
@@ -269,24 +269,31 @@ public void onPartitionBecomeBootstrapFromOffline(String partitionName) {
 
     @Override
     public void onPartitionBecomeStandbyFromBootstrap(String partitionName) {
+      logger.info("Partition state change notification from Bootstrap to Standby received for partition {}",
+          partitionName);
```

Comment:
Do you want to use lower level log (eg trace)? and run shell script to set the proper log level for the host?

Reply:
nope, this will not overwhelm the log and is critical to debug state transitions which happen only once during startup or rare thereafter